src/main/kotlin/com/wutsi/application/shell/endpoint/settings/screen/SettingsScreen.kt

```diff
@@ -79,6 +79,7 @@ class SettingsScreen(
                                 separator = true,
                                 children = listOf(
                                     ListItem(
+                                        padding = 5.0,
                                         caption = "Personal",
```

Comment:
Because we want to support multiple language, this should come from a language file.
The `caption` and `subCaption` should come from the language file.
See as exemple [this](https://github.com/wutsi/wutsi-shell-bff/blob/master/src/main/kotlin/com/wutsi/application/shell/endpoint/settings/account/screen/LinkAccountMobileScreen.kt#L55)
Put the text in [/src/main/resources/messages.properties](https://github.com/wutsi/wutsi-shell-bff/blob/master/src/main/resources/messages.properties)

Reply:
got it